scripts/import_app_title_to_desktop_flie.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import re
+from pathlib import Path
+
+
+# Load and validate environment variables
+def load_env_config() -> tuple[Path, Path, str] | None:
```

Comment:
What's the reasoning for using the environment instead of command line arguments?